microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -32,7 +32,7 @@ When hosting the WebView2 control, your app has access to the following features
 
 <!-- maintenance notes: add table rows for any new h2 sections -->
 
-<!-- todo: if "Managing SmartScreen" is an h2 section, add a row for it above
+<!-- 112 todo: if "Managing SmartScreen" is an h2 section, add a row for it above
```

Comment:
probably add this row into above table, in this 112 PR

Reply:
Renamed "Tracking Prevention" section to "Security and Privacy", added the incoming link there as its own tab-set, deleted "Managing SmartScreen" section.